src/components/Navigator/NavigatorCard.vue

```diff
@@ -247,6 +247,8 @@ export default {
     };
   },
   computed: {
+    isSameRoute: ({ technologyPath, $route }) => (
```

Comment:
The name is a little ambiguous as to what the comparison is for "same".

```suggestion
    isTechnologyRoute: ({ technologyPath, $route }) => (
```

Reply:
fixed!